### ISSUE TITLE
Tile Menu Layout Bugs

### ISSUE THREAD

**Issue by NicoleYarroch** [smartdevicelink/generic_hmi#196]
# Bug Report
I have found the following issues with the tile layout for menus:
1. If the `menuName` is bigger than the tile, it spills over the tile's bounds. For example, in the screenshot below "Acceleration Pedal Position" is too big for the tile so the word "Position" is not inside the tile's layout. I would assume that the word "Position" should not be shown at all.
2. The arrow for the  template title has a black line around the arrow when the menu template layout is `TILES` but not for any other template or when the menu template layout is `LIST`. 

![Menu tiles -  generic hmi](https://user-images.githubusercontent.com/19915981/64180772-5a96d880-ce33-11e9-8789-e94465c71c0a.png)

**Generic HMI Version:** 0.6.1
**Testing with:** SDL_iOS Swift example app v.6.3

**Comment by iCollin** [comment]
@NicoleYarroch I was unable to reproduce the bolded title menu arrow, if you are still seeing this please provide me with the steps you take to reproduce this issue.

**Comment by NicoleYarroch** [comment]
@iCollin I am still experiencing the bolded title menu arrow.

1. I ran the Swift example application in the SDL_iOS library. 
1. I opened the menu and then open a submenu. I thought it might be a submenu issue but some of them have the bolded arrow but some do not. Also one of the `PerformInteractionChoiceSet` menus also has the bolded arrow.
1. The submenu that shows a bolded arrow is "Get All Vehicle Data". The "Show Perform Interaction Choice Set" menu item opens a PICS menu with a bolded arrow.

**Comment by iCollin** [comment]
@NicoleYarroch Thank you for the additional information.

Using the SDL iOS Example app I was able to reproduce the issue and it seems to be due to many classes using the `.st0` css property to stroke the SVG images. Even though the arrow in the titlebar's `st0` class had just a `fill` attribute, it was picking attributes such as `stroke:black;stroke-width:2;` from other instances of the class. I have changed the name of the css class in `0xFF.svg` to prevent this accidental stroking.

Changes are included in https://github.com/smartdevicelink/generic_hmi/pull/203
